include/zephyr/bluetooth/bluetooth.h

```diff
@@ -1569,6 +1569,19 @@ enum {
 
 	/** Only sync to packets with constant tone extension */
 	BT_LE_PER_ADV_SYNC_TRANSFER_OPT_SYNC_ONLY_CTE = BIT(3),
+
+	/**
+	 * The following 2 options are mutually exclusive. You should set
+	 * at most one of these options at once. If no options are set, PAST
+	 * packets will be synced to and sync reports will be generated without
+	 * duplicate filtering.
+	*/
```

Comment:
This information should be placed in the documentation of the relevant options. The way it is added here probably won't result in good doxygen documentation.

Reply:
Do you mean it should be above the definition of each bit?

Reply:
Yes. They could just reference each other, stating that the other bit shall not be set at the same time :)